src/definition/settings/SettingType.ts

```diff
@@ -7,4 +7,7 @@ export enum SettingType {
     SELECT = 'select',
     STRING = 'string',
     MULTI_SELECT = 'multiSelect',
+	// Renders an input of type 'password' in the form. IMPORTANT - the value will NOT be encrypted
+	// it will be treated as a password just on the screen
+    PASSWORD = 'password',
```

Comment:
```suggestion
    // Renders an input of type 'password' in the form. IMPORTANT - the value will NOT be encrypted
    // it will be treated as a password just on the screen
    PASSWORD = 'password',
```